src/children.js

```diff
@@ -3,21 +3,42 @@
  * @param {object | object[]} node or nodes
  * @returns {object[]}
  */
-export default function children (node) {
+export function get (node) {
+	return getDetails(node).map(({node}) => node);
+}
+
+/**
+ * Get a node's children and the corresponding properties and indices
+ * @param {object | object[]} node or nodes
+ * @returns {object[]}
+ */
+export function getDetails (node) {
```

Comment:
The more I think about it, the more I think `getPaths` might be a better name than `getDetails`.

Reply:
Yes, we can decide on the correct terminology that we'll use consistently across the codebase as you mentioned in #46. I think paths is pretty good, but wondering if there's a better term